### DISCUSSION TITLE
Placing an element with 0 margins/gap

### DISCUSSION THREAD

**Discussion by adarshpunj** [zauberzeug/nicegui#3419] [Q&A] [answered]
### Question

I'm trying to remove the left/right margins in the viewport. I'm trying to recall how or if I ever did this before but no luck.
```
with ui.row().classes("gap-0 w-full"):
    # some element
```
Can someone help me on this? Thanks.

**Comment by rodja** [comment, marked as answer]
Maybe https://nicegui.io/documentation/query#modify_default_page_padding is what you are looking for?

**Reply by adarshpunj** [reply]
Yes, exactly. Thanks!
